### PR TITLE
Fix/fix remaining issues

### PR DESCRIPTION
**Pull request by Vietprole**
Rewrite README.md.
Style header and footer using class.
Fix input not working.
Move footer links to list.
Wrap img inside a div.

### CODE REVIEW

File: src/css/sections/footer.css [hunk 1]
```diff
@@ -1,4 +1,4 @@
-footer {
+.main-footer {
```
Comment (hoangle-agilityio) [inline src/css/sections/footer.css:1]: footer is enough

File: src/css/sections/header.css [hunk 2]
```diff
@@ -1,7 +1,8 @@
-header {
+.main-header {
```
Comment (hoangle-agilityio) [inline src/css/sections/header.css:1]: `header` is enough

Comment (Vietprole) [reply src/css/sections/header.css]: Ok, I have renamed it

File: src/css/sections/footer.css (hunk 1 above)
Comment (Vietprole) [reply src/css/sections/footer.css]: Ok, I have renamed it

**Pull request merged**